server/utils/auth/command_gate.py

```
@@ -142,6 +142,62 @@ def gate_command(
+        user_id = getattr(state, "user_id", None)
+        with db_pool.get_connection() as conn:
+            with conn.cursor() as cur:
+                set_rls_context(cur, conn, user_id, log_prefix="[Gate:refresh_perms]")
+                cur.execute(
+                    "SELECT tool_key FROM org_tool_permissions WHERE org_id = %s AND enabled = true",
+                    (org_id,),
+                )
+                state.permitted_tools = {row[0] for row in cur.fetchall()}
+        rc.delete(dirty_key)
+    except Exception:
+        pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)